test/mochiweb_socket_server_tests.erl

```diff
@@ -38,6 +38,9 @@ client_fun(Socket, [{send_pid, To} | Cmds]) ->
     To ! {client, self()},
     client_fun(Socket, Cmds);
 client_fun(Socket, [{send, Data, Tester} | Cmds]) ->
+    client_fun(Socket, [{send, Data, Tester, 0} | Cmds]);
+client_fun(Socket, [{send, Data, Tester, Delay} | Cmds]) ->
+    timer:sleep(Delay),
```

Comment:
It's not good to have tests that sleep, they can cause the test runs to take much longer than they should and they can intermittently pass or fail depending on external factors (e.g. the specs or load on the system running the tests). It's best to have code that doesn't require the wall clock to work, or at least use mocking when testing it to avoid any sleeping.